test/unit/source/raster_dem_tile_worker_source.test.js

```diff
@@ -11,7 +10,7 @@ test('loadTile', (t) => {
         source.loadTile({
             source: 'source',
             uid: 0,
-            rawImageData: new RGBAImage({height: 8, width: 8}, new Uint8ClampedArray(256)),
+            rawImageData: {data: new Uint8ClampedArray(256), width: 8, height: 8},
```

Comment:
Nit: Can we also have a test for the other case (`window.imgeBitmap`)?

Reply:
I tried, but JSDom (or at least the version we use) doesn't provide `OffscreenCanvas` or `createImageBitmap`, and mocking it fully inside the test feels too fragile. Not sure how to proceed otherwise.

Reply:
Gotcha. Since this would be caught before a release by manual testing, I think it's ok to do without the automated test.